outlook/src/classes/Partner.ts

```diff
@@ -28,25 +37,82 @@ class Partner {
     }
 
     getInitials() : string {
-        if (!this.name) {
-            return "";
-        }
-        const names = this.name.split(" ");
-        let initials = names[0].substring(0, 1).toUpperCase();
-        
-        if (names.length > 1) {
-            initials += names[names.length - 1].substring(0, 1).toUpperCase();
-        }
-        return initials;
+
+        let rgx = new RegExp(/(\p{L}{1})\p{L}+/, 'gu');
```

Comment:
I bet this require some explanation ...

Reply:
When looking for initials, this is done to only catch words having a length > 2 and which only contain letters. When we have a displayed name of "John Doe (something)", this will enable the method to return "JD" and prevents it from returning something like "J(".

A comment was added to explain this.